docs/development/adding-a-package-manager.md

```diff
@@ -111,81 +111,126 @@ Use `updateLockedDependency` to directly update dependencies in lock files.
+
+There are two ways to update lock files:
 
-There are two approaches which can be used:
+- Call a command like `<tool> update` to update all locked dependencies (plus transitive dependencies), updating the whole lock file where possible
+- Delete the lock file, then call a command like `<tool> install` to create a new lockfile
 
-- Delete the existing lock file, then call a command like `<tool> install` to regenerate it, or
-- Call a command like `<tool> update` if such a command exists to satisfy this same requirement (updating the entire lock file where possible)
+If you can, use the `<tool> update` method, as that keeps platform-specific information.
 
-Where available, the second approach is better because lock file may sometimes have platform-specific information (e.g. amd64, arm64) which can be lost if the lock file is regenerated completely as in the first approach.
+#### Keep platform-specific information
+
+Lock files may have platform-specific information (e.g. `amd64`, `arm64`).
+If you delete the lock file, and then create a new lock file with `<tool> install`, the platform-specific information is lost.
+
+If you can, use the `<tool> update` method instead!
 
 ### Lock file updating after a package file change
 
-This functionality is often mandatory from initial implementation.
+Updating lock files after a package file changes is a fundamental feature.
+This means you often need to build it first, when adding new package manager to Renovate.
 
-In this scenario, an `updateArtifacts()` function must be added.
-Its purpose is to essentially "sync" the lock file to the package file changes made by Renovate, so that both files can be updated in the same commit.
+Add a `updateArtifacts()` function, that "syncs" the lock file to the package file changes made by Renovate.
+This way, both files can be updated in the same commit.
 
 Usually, the flow is like this:
 
-1. Renovate makes changes to the version or constraint in the package file directly,
-2. Renovate calls a tool command like "<tool> install", "<tool> lock", etc.
-3. If the tool command resulted in a changed lock file (it usually should), then Renovate commits the changes along with the package file change
+1. Renovate directly changes the version or constraint in the package file,
+2. Renovate calls a tool command like `<tool> install`, `<tool> lock`, etc.
+3. If the tool command changes the lock file (which it usually should!), then Renovate commits the changed lock file and the package file
 
 ### Locked version extracting and dependency pinning
 
-The next step is for the manager's "extract" functionality to return a `lockedVersion` for dependencies whenever a lock file exists.
-To do this, the manager should:
+The next step is for the manager's "extract" feature to return a `lockedVersion` for dependencies, whenever a lock file exists.
+To do this, the manager must:
```

Comment:
```suggestion
To do this, the manager MUST:
```

Reply:
I don't think this needs to be so strong